app/views/checkouts/index.html.erb

```diff
@@ -4,72 +4,74 @@
+        </a>
 
-      <ul class="dropdown-menu" aria-labelledby="dropdownMenuLink">
-        <li class="dropdown-item" data-sort="status">Status</li>
-        <li class="dropdown-item active" data-sort="due_date">Due date</li>
-        <li class="dropdown-item" data-sort="title">Title</li>
-        <li class="dropdown-item" data-sort="author">Author name</li>
-        <li class="dropdown-item" data-sort="call_number">Call number</li>
-      </ul>
+        <ul class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+          <li class="dropdown-item" data-sort="status">Status</li>
+          <li class="dropdown-item active" data-sort="due_date">Due date</li>
+          <li class="dropdown-item" data-sort="title">Title</li>
+          <li class="dropdown-item" data-sort="author">Author name</li>
+          <li class="dropdown-item" data-sort="call_number">Call number</li>
+        </ul>
+      </div>
     </div>
-  </div>
 
-  <% if other_checkouts.any?(&:renewable?) %>
-    <%= link_to all_eligible_renewals_path(group: params[:group]), class: 'btn btn-info mb-3', method: :post do %>
-      <%= sul_icon :'renew' %> Renew <%= pluralize(other_checkouts.count(&:renewable?), 'eligible item') %>
+    <% if other_checkouts.any?(&:renewable?) %>
+      <%= link_to all_eligible_renewals_path(group: params[:group]), class: 'btn btn-info mb-3', method: :post do %>
+        <%= sul_icon :'renew' %> Renew <%= pluralize(other_checkouts.count(&:renewable?), 'eligible item') %>
+      <% end %>
     <% end %>
-  <% end %>
 
-  <div class="d-none d-md-flex row font-weight-bold list-header">
-    <div class="row col-md-4">
-      <div class="col-md-6" data-sort="status">Status</div>
-      <div class="col-md-6 active" data-sort="due_date">Due date</div>
-    </div>
-    <div class="row col-md-8">
-      <div class="col-md-7" data-sort="title">Title</div>
-      <div class="row col-md-5">
-        <div class="col-md-6" data-sort="author">Author</div>
-        <div class="col-md-6 call_number" data-sort="call_number">Call number</div>
+    <div class="d-none d-md-flex row font-weight-bold list-header">
+      <div class="row col-md-4">
+        <div class="col-md-6" data-sort="status">Status</div>
+        <div class="col-md-6 active" data-sort="due_date">Due date</div>
+      </div>
+      <div class="row col-md-8">
+        <div class="col-md-7" data-sort="title">Title</div>
+        <div class="row col-md-5">
+          <div class="col-md-6" data-sort="author">Author</div>
+          <div class="col-md-6 call_number" data-sort="call_number">Call number</div>
+        </div>
       </div>
     </div>
+    <ul class="list checkouts list-group">
+      <%= render other_checkouts %>
+    </ul>
   </div>
-  <ul class="list checkouts list-group">
-    <%= render other_checkouts %>
-  </ul>
-</div>
-
+<% else %>
+  <h2>Checked out: <%= @checkouts.length %><%= " (#{patron_or_group.remaining_checkouts} remaining)" if patron_or_group.remaining_checkouts %></h2>
```

Comment:
I think we still need the `div.page-section` here in order to have appropriate spacing between this and the policies. I see you took a slightly different approach to this in fines and requests, so maybe it's worth just doing the same thing here (putting the `if` condition around the sort + data instead of the whole thing)?

Reply:
We threw in the a `div.page-section` into the `else` branch. The approach is slightly different here because we don't display `Recalled: 0` (whether they have checkouts or not) if a user has no recalled items.

Reply:
👍 WFM.